lib/bucky/core/test_core/test_class_generator.rb

```diff
@@ -15,9 +15,9 @@ module TestClassGeneratorHelper
 
         def add_test_procedure(procedures)
           procedures.each.with_index(1) do |procedure, index|
-            puts "  #{index}:#{procedure[:proc]}" if procedure.key?(:proc)
+            puts procedure[:proc].insert(0, "  #{index}:") if procedure.key?(:proc)
```

Comment:
In the old code. procedure[:proc] just store the proc name.
But I want to use procedure[:proc] in the error message.
It will be clearly to show the proc index number in error message, because that some procs got same name in the case. 
So I just insert the index into procedure[:proc].

Reply:
I got it!